user_test.go

```diff
@@ -0,0 +1,271 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestLoginReadsBody(t *testing.T) {
+	aWebUser := &webUser{
+		Username: "Username",
+		Password: "Password",
+	}
+
+	aDBUser := &dbUser{
+		Username: aWebUser.Username,
+		Password: aWebUser.Password,
+	}
```

Comment:
Initializing `aDBUser` her might be unnecessary

Reply:
Taken care of. While aDBUser needs to be remained declared to prevent nil pointer dereference errors vis-à-vis the necessary gorm queries, you are correct that the initialization of the Username and Password values are entirely unnecessary for this test.